redaxo/src/core/boot.php

```diff
@@ -120,6 +120,17 @@
     }
 }
 
+// ----------------- HTTPS REDIRECT
+if (rex::getProperty('use_https') === true || rex::getProperty('use_https') === rex::getEnvironment()) {
+    if (!rex_request::isHttps() && !rex::isSetup()) {
```

Comment:
Wir müssen hier explizit die Console noch ausschließen. Wenn use_https=true, funktioniert die gerade gar nicht mehr.
Am besten per `PHP_SAPI` ausschließen, nicht nur für die symfony-console (`rex::getConsole()`).

Reply:
> Wenn use_https=true, funktioniert die gerade gar nicht mehr.

Mit `rex::getProperty('use_https') === true` sollte es doch bei true funktionieren oder?

Mit `PHP_SAPI` meinste wahrscheinlich `php_sapi_name() != 'cli'`

Reply:
https://github.com/redaxo/redaxo/blob/ce937226dd406f143412f24cef304bc4647151d3/redaxo/src/addons/cronjob/bin/run#L7

Reply:
> Mit rex::getProperty('use_https') === true sollte es doch bei true funktionieren oder?

Bei mir scheint er in er Console dann in den Redirect zu gehen, und durch exit() wird dann abgebrochen. Der eigentliche Command wird nicht ausgeführt.

<img width="530" alt="screenshot 2018-05-19 14 49 59" src="https://user-images.githubusercontent.com/330436/40268758-eafa020a-5b73-11e8-9683-8c184b38cc21.png">

Reply:
@gharlan Ah sorry, hatte nicht richtig gelesen, dachte mit `true` funktionierte die Umleitung im Browser nicht mehr.

Ok, `php_sapi_name()` und `PHP_SAPI` machen das gleiche.
Fix kommt.